base/windows/CharsStatusWindow.js

```diff
@@ -153,10 +159,7 @@ export class CharsStatusWindow {
             let char = party_data.members[i];
             current_chars.push(char.key_name);
             let info_sprite = this.info_sprites[char.key_name];
-            if (!info_sprite.visible) {
-                this.toggle_char_info(info_sprite);
-                info_sprite.visible = true;
-            }
+            info_sprite.group.visible = true ? true : false;
```

Comment:
`info_sprite.group.visible = !info_sprite.group.visible` should be enough. (are you checking true?)

Reply:
This will fail if it's already true upon starting the cycle though. We want it to always be true

Reply:
I did actually make a mistake though, I will correct it

Reply:
But here you're doing something like `if (true) ...`. `info_sprite.group.visible = true;` should be fine, then.

Reply:
Check [this commit](https://github.com/jjppof/goldensun_html5/pull/16/commits/2de13248f049bc0f27be40e567546c40e41c2811) please